modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCTransactionStateTest.java

```diff
@@ -64,7 +64,8 @@ public void setUp() {
             public Object answer(InvocationOnMock invocation) {
                 Object[] arguments = invocation.getArguments();
                 LogRecord logRecord = (LogRecord) arguments[0];
-                if (logRecord.getLevel() == Level.WARNING) {
+                if (logRecord.getLevel() == Level.WARNING
+                        && !logRecord.getSourceMethodName().equals("finalize")) {
```

Comment:
that is a pretty cheap fix! But I agree a finalizer warning is a little off topic for this test.